plc4j/drivers/modbus/src/test/java/org/apache/plc4x/java/modbus/base/optimizer/LittleEndianByteSwapTest.java

```diff
@@ -248,7 +249,8 @@ public void testLittleEndianByteSwap() throws Exception {
         Map<PlcReadRequest, BaseOptimizer.SubResponse<PlcReadResponse>> readResponses = new HashMap<>();
         readResponses.put(firstRequest, new BaseOptimizer.SubResponse<>(
             new DefaultPlcReadResponse(firstRequest, Map.of(
-                "coils0", new DefaultPlcResponseItem<>(PlcResponseCode.OK, new PlcRawByteArray(Hex.decodeHex("3060480c00c084000000000000000000000000000000")))))));
+                //"coils0", new DefaultPlcResponseItem<>(PlcResponseCode.OK, new PlcRawByteArray(Hex.decodeHex("3060480c00c084000000000000000000000000000000")))))));
+                "coils0", new DefaultPlcResponseItem<>(PlcResponseCode.OK, new PlcRawByteArray(Hex.decodeHex("10404004004004000000000000000000000000000044")))))));
```

Comment:
I assume the reason the data here changed, is because of the changed endianess in the connection string, right? In that case would I prefer us to have different tests for different encodings. Now it's impossible to switch between different types of PLCs ... in order to switch back to mine (if I updated the data in them) I would still need to change this code.

Reply:
I used the bit pattern from the optimizer test, 175 coils, on two different devices and they returned the same byte sequence and according to the Modbus standard, so I used it as a reference.
I want to validate with other devices, but bit (coil) requests are always answered in BIG_ENDIAN, which is consistent with the standard, but I need to reconfirm this.